src/test/lido/ObolLidoSplit.t.sol

```diff
@@ -201,6 +195,6 @@ contract ObolLidoSplitTest is ObolLidoSplitTestHelper, Test {
 
     assertEq(ERC20(WSTETH_MAINNET_ADDRESS).balanceOf(fuzzFeeRecipient), expectedFee, "invalid fee transferred");
 
-    assertEq(ERC20(WSTETH_MAINNET_ADDRESS).balanceOf(anotherSplit), wstETHDistributed - expectedFee, "invalid amount");
+    assertGe(ERC20(WSTETH_MAINNET_ADDRESS).balanceOf(anotherSplit), wstETHDistributed - expectedFee, "invalid amount");
```

Comment:
There is off-by-one error due to integer division above: `uint256 expectedFee = (wstETHDistributed * fuzzFeeShare) / PERCENTAGE_SCALE;`
The difference is exact one wei for the first fuzzing pass.